pom.xml

```diff
@@ -421,9 +443,11 @@
                 </exclusions>
             </dependency>
             <dependency>
+                <!-- gt-imageio-ext-gdal is 'works with' so it must stay in provided scope -->
                 <groupId>org.geotools</groupId>
                 <artifactId>gt-imageio-ext-gdal</artifactId>
                 <version>${gt.version}</version>
+                <scope>provided</scope>
```

Comment:
this had transitive dependencies on the imageio-ext things, and this made it all simpler.